pkg/webhook/controlplaneexposure/ensurer_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Ensurer", func() {
 			checkETCDMain(etcd)
 		})
 
-		It("should modify existing elements of etcd-main statefulset", func() {
+		It("should modify existing elements of etcd-main statefulset which are not 80Gi", func() {
```

Comment:
```suggestion
		It("should modify etcd storage config when etcd is new and specifies a storage config", func() {
```

The code and tests here have been copied over from g/g 4 years ago, when etcd was being as statefulset directly by g/g code (druid did not exist back then). So the test specs also talk about statefulsets rather than `Etcd` resources.

Can you please make corresponding changes to all the test spec names in this file?
- `should modify etcd-main storage config when etcd-main is new and does not specify a storage config`
- `should modify etcd-main storage config when etcd-main is new and specifies a storage config`
- `should not modify etcd-main storage config when etcd-main is old`
- `should always modify etcd-events storage config`

Reply:
Addressed in [9eab017](https://github.com/gardener/gardener-extension-provider-aws/pull/933/commits/9eab0176574b78dac948e23b12e4c50148b3669c).